src/content/pages/beginners-day.mdx

```diff
@@ -39,18 +37,32 @@ Whether you're considering a career change, looking for your first tech role, or
+* How to install Python and use virtual environments
+* How to install Python packages
+* Python basics like variables, loops, functions, and conditionals
+* How to use Django to build a simple web app and deploy it to a live server
+* HTML and CSS fundamentals
+* Version control with Git and GitHub.
+
+Just bring a laptop, and by the end of the day, you'll have deployed your first web application to the internet!
 
-We are also looking for **mentors** for Django Girls. Applications for mentors close on May 18th, and successful mentors will be informed by May 25th.
+We are also looking for **mentors** for Django Girls. Applications for mentors close on **July 4th**.
 
 <Button url="https://docs.google.com/forms/d/e/1FAIpQLScJeKrKmDmS8BNpcwxeEmqkeMbJvtCSyhskxGMZnba5yGYv7Q/viewform?usp=header">Become a Django Girls Mentor!</Button>
 
 ## Humble Data
 
-Humble Data is an inclusive, beginner-friendly workshop that introduces participants to data science using Python. It is aimed at people from underrepresented groups in tech but is open to everyone. Throughout this interactive session, you'll learn essential Python programming concepts while exploring real-world datasets, creating visualizations, and discovering patterns through analysis. Experienced mentors will guide you through each step, from setting up your environment to implementing data analysis techniques. The workshop emphasizes a hands-on approach where you'll actively work with tools like Jupyter notebooks, pandas, and matplotlib in a supportive, collaborative atmosphere.
+Humble Data is an inclusive, beginner-friendly workshop that introduces participants to data science using Python. It is aimed at people from underrepresented groups in tech but is open to everyone. Throughout this interactive session, you'll learn essential Python programming concepts while exploring real-world datasets, creating visualizations, and discovering patterns through analysis. Experienced mentors will guide you through each step, from setting up your environment to implementing data analysis techniques.
+
+This workshop is also designed for people with no previous knowledge of Python or data science. During the workshop, attendees will learn:
+* Python basics like variables, loops, functions, and conditionals
+* How to read in data using Python
+* How to manipulate and clean data using pandas
+* How to visualise data using Matplotlib and Seaborn.
 
-No prior programming or data science experience is necessary: just bring your laptop and curiosity, and you'll leave with practical skills to begin your data science journey.
+Just bring a laptop, and you'll leave with practical skills to begin your data science journey.
 
-We are also looking for **mentors** for Humble Data. Applications for mentors close on May 18th, and successful mentors will be informed by May 25th.
+We are also looking for **mentors** for Django Girls. Applications for mentors close on **July 4th**.
```

Comment:
The link below is to the Humbe Data form, and it's the HD section. this probably shouldn't be changed.